### PR TITLE
ID-3571 [FIX] Fixes form components' width to match container size

### PR DESCRIPTION
**Pull request by armine-fliplet**
### Result:

https://github.com/Fliplet/fliplet-widget-form-builder/assets/128052346/ad4b87ee-a089-4eb8-9003-80b4f439e365

### CODE REVIEW

File: css/form.css [hunk 1]
```diff
@@ -1524,6 +1524,10 @@ form hr {
     display: flex !important;
     align-items: center;
   }
+
+  .scanner.btn.btn-primary {
+    margin-bottom: 8px;
```
Comment (tonytlwu) [inline css/form.css:1529]: Why is this here? Doesn't seem related to custom field widths.

Comment (armine-fliplet) [reply css/form.css]: Removed!

File: css/form.css [hunk 2]
```diff
@@ -6,6 +6,7 @@
   display: flex;
   flex-direction: column;
   margin: 0;
+  margin-bottom: 15px;
```
[the remark this answers is not in the document]
Comment (armine-fliplet) [reply css/form.css:9]: I have changed the grid gap from `px` to `%` and it's showing too small size for the row gap. That's why I set the gap only for the columns and added margin-bottom for all fields.

**Review by tonytlwu — APPROVED** [review]
@inna-bieshulia This is ready for testing

Skip compile: 
Includes translation: 
Initiative: Custom field width
Impact: High
Affected: All form instances
Verify: Forms are laid out on the page as expected
